src/pattern-library/components/patterns/DialogPatterns.js

```diff
@@ -189,6 +289,34 @@ export default function DialogPatterns() {
         </div>
       </Pattern>
 
+      <Pattern title="Modal with long content">
+        <div className="Example">
+          <p>
+            Modals that may contain a lot of content may need to handle overflow
+            (i.e. make their content scrollable) so that the modal height
+            doesn&apos;t exceed available viewport space.
+          </p>
+          <div className="Example__content">
+            <div className="Example__usage">
+              <p>
+                To make something in a modal scroll-able, apply{' '}
+                <code>overflow: auto</code> to the element you wish to contain.
+                This element needs to be an immediate-child element of the{' '}
+                <code>Modal</code>.
+              </p>
```

Comment:
This isn't quite right, argh. I'll fix after getting feedback on the overall approach.